### ISSUE TITLE
Showcase links to our key web assets

### ISSUE THREAD

**Issue by neontapir** [agiledenver/agiledenver.github.io#3]
- Meetups
  - SIGs once reorg is complete
- AD LinkedIn group
- conference web site (last or
  next MHA) 
  - maybe past archive
- Yahoo Groups (maybe?)
- Flickr (maybe?)

We have these by virtue of picking Jekyll and GitHub pages:
- AD Twitter
- AD GitHub org or key repos (probably not yet?)

**Comment by neontapir** [comment]
Duplicate of #2
